.CI/build_all

```diff
@@ -1,28 +1,54 @@
 #!/usr/bin/env bash
 
 
+# Fast finish the PR.
+echo ""
+echo "Checking to see if this build is outdated."
+.CI/travis_fast_finish.py || { echo "Failing outdated build to end it."; exit 1; }
+
 # Set the numpy variable. This isn't used, but conda-build complains if we haven't set it already.
 export CONDA_NPY=19
 
 # Remove homebrew.
-brew remove --force --ignore-dependencies $(brew list)
-brew cleanup -s
-rm -rf $(brew --cache)
+echo ""
+echo "Removing homebrew from Travis CI to avoid conflicts."
+curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall > ~/uninstall_homebrew.sh
+chmod +x ~/uninstall_homebrew.sh
+~/uninstall_homebrew.sh -fq
+rm ~/uninstall_homebrew.sh
+
+# Install Miniconda.
+echo ""
+echo "Installing a fresh version of Miniconda."
+[ ! -f ~/cache/miniconda.sh ] && curl -L https://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh > ~/cache/miniconda.sh
+bash ~/cache/miniconda.sh -b -p ~/miniconda
+source ~/miniconda/bin/activate root
+
+# Reuse packages from the cache.
+mkdir -p ~/cache/pkgs
+rm -rf ~/miniconda/pkgs
+mv ~/cache/pkgs ~/miniconda/pkgs
 
-# Install and configure conda environment.
-curl -L -O https://raw.githubusercontent.com/conda-forge/conda-smithy/e976c7e84bb3c4846e7562afd1a42c4b535b51f5/bootstrap-obvious-ci-and-miniconda.py
-python bootstrap-obvious-ci-and-miniconda.py ~/miniconda x64 3 --without-obvci && source ~/miniconda/bin/activate root
+# Configure conda.
+echo ""
+echo "Configuring conda."
 conda config --add channels conda-forge
 conda config --set show_channel_urls true
 conda install --yes --quiet conda-build-all=1.0.0
 conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
+# Add standard packages back to the cache.
+mkdir -p ~/cache/pkgs
+cp -r ~/miniconda/pkgs ~/cache
+
```

Comment:
Awesome, I did this for a repo the other day and was wondering if we were using it here! :-)

(We can do the same on the other CIs, even if they are not the bottleneck)